resources/service.rb

```diff
@@ -57,12 +58,14 @@ def create_init
       template "/etc/systemd/system/#{svc_name}.service" do
         source 'init_systemd.erb'
         cookbook 'snort'
+        variables resource: new_resource
```

Comment:
I'd prefer to explicitly pull in `new_resource.options` here so it's clearer on the other end and I don't see the benefit of pushing attribute selection into the template. I mean yes it's pretty flexible, but as a rule I tend/try to keep as much stuff out of templates.